ext/standard/exec.c

```diff
@@ -272,8 +269,7 @@ PHP_FUNCTION(passthru)
    Escape all chars that could possibly be used to
    break out of a shell command
 
-   This function emalloc's a string and returns the pointer.
-   Remember to efree it when done with it.
+   This function returns a zend_string, remember to release it when done.
```

Comment:
```suggestion
   This function returns a newly allocated zend_string, remember to release it when done.
```

Reply:
We could call it "owned", although that term is a bit ambiguous for RC. An alternative would be "addref'd".

Reply:
I think owned is decent, addref'd is worse imo because interned strings don't get addref'd.

Reply:
Done